File: checksum/Byte-Stuffing.py
def byte_stuffing(input):

  flagByte = "FLAG"
  escapeByte = "E"

  x = input.replace(escapeByte, escapeByte*2)
  y = x.replace(flagByte, escapeByte+flagByte)

  return print (f"Frame: {flagByte}{y}{flagByte}")

byte_stuffing("EEBZ")

File: checksum/byteStuffing.go
package main

import (
	"fmt"
	"strings"
)

func byteStuffing(input string) {
	flagByte := "FLAG"
	escapeByte := "E"

	x := strings.ReplaceAll(input, escapeByte, escapeByte+escapeByte)
	y := strings.ReplaceAll(x, flagByte, escapeByte+flagByte)

	fmt.Printf("Frame: %s%s%s\n", flagByte, y, flagByte)
}

func main() {
	byteStuffing("EEBZ")
}
